### ISSUE TITLE
🐛 [Bug] Misspelled “their” of profile::invites::description

### ISSUE THREAD

**Issue by Tirokk** [Human-Connection-Community/Human-Connection-New-20201003#2230]
> [<img alt="infinite-dao" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/infinite-dao) **Authored by [infinite-dao](https://github.com/infinite-dao)**
_<time datetime="2019-11-13T08:38:06Z" title="Wednesday, November 13th 2019, 9:38:06 am +01:00">Nov 13, 2019</time>_
_Closed <time datetime="2019-11-19T23:19:49Z" title="Wednesday, November 20th 2019, 12:19:49 am +01:00">Nov 20, 2019</time>_
---

## :bug: Bugreport
In there is a misspelled “thier” in https://github.com/Human-Connection/Human-Connection/blob/a0fe9cb94cb7d814b42dc46c48bdcd432e676e51/webapp/locales/en.json#L174 and also the “their” seems to refer to the very “somebody” in the title 
https://github.com/Human-Connection/Human-Connection/blob/a0fe9cb94cb7d814b42dc46c48bdcd432e676e51/webapp/locales/en.json#L173 
I wonder about the “their” but I learned that “their” referring to “somebody/someone” is just English (see https://forum.wordreference.com/threads/never-sit-on-someone-elses-blanket-without-their-permission.2245981/). In German is a discussion in [lokalise.com](https://lokalise.com/project/556252725c18dd752dd546.13222042/?view=single&reference_lang_id=640&single_lang_id=666) to term it more neutral, e.g. “Enter the e-mail address for invitation” or so. But we can leave it at “their” and just correct the misspelled word.

- Are there multiple e-mails to be inserted or just one?

**Comment by Tirokk** [comment]
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-11-13T09:09:17Z" title="Wednesday, November 13th 2019, 10:09:17 am +01:00">Nov 13, 2019</time>_

---


Hey @infinite-dao, definitely a typo that can be fixed. Their, in English is a good gender neutral pronoun, so we'd like to keep it.
Thanks for spotting this!

**Comment by Tirokk** [comment]
> [<img alt="infinite-dao" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/infinite-dao) **Authored by [infinite-dao](https://github.com/infinite-dao)**
_<time datetime="2019-11-13T09:21:44Z" title="Wednesday, November 13th 2019, 10:21:44 am +01:00">Nov 13, 2019</time>_

---


OK, right. And the answer to my final question? ;-)

**Comment by Tirokk** [comment]
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-13T12:25:18Z" title="Wednesday, November 13th 2019, 1:25:18 pm +01:00">Nov 13, 2019</time>_

---


Please check also the German translation. It uses the term 'seine' meaning 'his' which may exclude female users.

**Comment by Tirokk** [comment]
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-13T12:31:07Z" title="Wednesday, November 13th 2019, 1:31:07 pm +01:00">Nov 13, 2019</time>_

---


I also think we should have a consistent usage of the term URL. Sometimes it is in lowercase, sometime in uppercase.

**Comment by Tirokk** [comment]
> [<img alt="infinite-dao" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/infinite-dao) **Authored by [infinite-dao](https://github.com/infinite-dao)**
_<time datetime="2019-11-13T13:19:33Z" title="Wednesday, November 13th 2019, 2:19:33 pm +01:00">Nov 13, 2019</time>_

---


> Please check also the German translation. It uses the term 'seine' meaning 'his' which may exclude female users.

OK, exclude female users yes and no. Interesting maybe is the [German Youtube talk of „Dr. Tomas Kubelik: Wie Gendern unsere Sprache verhunzt“ (45min)](https://www.youtube.com/watch?v=Ri-kVYDTEAk) and I concur with Tomas Kubelik, that our German language uses “male” grammar, if you do not know the sex of a person, eg. „Die Polizei, Dein Freund und Helfer“ instead of „Die Polizei, Deine Freundin und Helferin“ ;-) 

Anyway I tend to translate the correct English “…enter their e-mail…” to German „…die E-Mail …“ Or as we will have a German title wiht something like „_Lade jemanden zu Human Connection ein_“ may be followed by
- „_Zur Einladung dessen E-Mail-Adresse eintragen_“ ?
- „_Zur Einladung gib jemandes E-Mail-Adresse ein_“ to reuse the Genetiv or so ;-)

**Comment by Tirokk** [comment]
> [<img alt="Mogge" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Mogge) **Authored by [Mogge](https://github.com/Mogge)**
_<time datetime="2019-11-13T18:16:44Z" title="Wednesday, November 13th 2019, 7:16:44 pm +01:00">Nov 13, 2019</time>_

---


@infinite-dao I agree with what you are saying. Aber ich finde "gebe SEINE E-Mail Adresse ein" wirklich sehr maskulin. When I checked today I saw it already changed. Now in lokalize it is "Zur Einladung die E-Mail-Adresse hier eintragen.", which is fine. And thanx for the link, I will take a look at it.
